bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -3,7 +3,7 @@ argh=0.26.*
 beautifulsoup4=4.6.*
 conda=4.3.33
 conda-build=2.1.18
-galaxy-lib=17.9.*
+galaxy-lib >=18.5.4,<19.0a0
```

Comment:
`galaxy-lib 18.5.4` includes the `CONDA_IMAGE` configurability.
Do we need to care whether `pip` is fine with this syntax? As is, this is only used as input for `conda install --file`.

Reply:
Although technically unrelated, I increased the lower bound to `18.5.5` so that we can test out if https://github.com/galaxyproject/galaxy-lib/pull/100 works.